### DISCUSSION TITLE
L&S Emotion Home Base doesn't work with ZHA

### DISCUSSION THREAD

**Discussion by isabelkort** [zigpy/zha-device-handlers#2747] [Q&A]
I connected the Light with L&S EMotion Home Base with ZHA. It looks good, but if i turn on the Light over ZHA, the Home Base light glows red.
Does anyone have an idea what I can do?

**Comment by Hedda** [comment]
It a device is not showing all expected entities under ZHA then it probably needs a quirk (ZHA Device Handler) so do submit a new "Device support request" issue under Issues -> https://github.com/zigpy/zha-device-handlers/issues/

Note that other than device signature you should also describe setup/environment exactly and all involved devices in much greater detail.

PS: Edit ZHA docs if unclear -> https://www.home-assistant.io/integrations/zha#how-to-add-support-for-new-and-unsupported-devices

**Comment by LordDoggo** [comment]
I have the same problem. Did you find any solution to this?

**Reply by isabelkort** [reply]
Unfortunately not, but I haven't tested it any further in a while.

**Reply by LordDoggo** [reply]
Okay, sad. Thank you for that fast answer.

**Reply by isabelkort** [reply]
let me know if you have a solution to the problem

**Comment by Hedda** [comment]
Suggest that you submit a new and updated [Device Support Request] similar to this old one:

* https://github.com/zigpy/zha-device-handlers/issues/1244 

Be sure to that you run latest Home Assistant and include all the information and logs required:

*  https://github.com/zigpy/zha-device-handlers/issues/new/choose

PS: Again, please help improve ZHA documentation if this procedure is unclear:

* https://www.home-assistant.io/integrations/zha#how-to-add-support-for-new-and-unsupported-devices
